README.md

```diff
@@ -1925,6 +1925,25 @@ the list of flags you return from your `FlagsForFile` function in your
 
 See [issue #303][issue-303] for details.
 
+### Install YCM with [NeoBundle][NeoBundle]
+[NeoBundle][NeoBundle] can do the compilation for you just add in your vimrc:
+
+    NeoBundle 'Valloric/YouCompleteMe', {
+         \ 'build'      : {
+            \ 'mac'     : './install.sh --clang-completer --system-libclang --omnisharp-completer',
+            \ 'unix'    : './install.sh --clang-completer --system-libclang --omnisharp-completer',
+            \ 'windows' : './install.sh --clang-completer --system-libclang --omnisharp-completer',
+            \ 'cygwin'  : './install.sh --clang-completer --system-libclang --omnisharp-completer'
+            \ }
+         \ }
+
+But you can have a issue about the time needed to get the sub modules and
+compile the whole thing, on my Lenovo t440s it take near 15 minutes, so to
```

Comment:
Period after "compile the whole thing."

Then drop the part about your machine and just have:

"To increase the Neobundle timeout to 1500 seconds, add the following to your vimrc:"